ui/src/ui/components/InputField/InputField.tsx

```diff
@@ -0,0 +1,74 @@
+// Import packages
+import React, { useState } from 'react';
+
+// Import react icon type
+import type { IconType } from 'react-icons';
+
+// Import css
+import './InputField.css';
+import '../../../App.css';
+
+// Defining prop type
+type Props = {
+  title: string;
+  id: string;
+  type: string;
+  value?: string;
+  placeHolder: string;
+  Icon: IconType;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  disabled: boolean;
+};
+
+export default function InputField({
+  title,
+  id,
+  type,
+  onChange,
+  value,
+  placeHolder,
+  Icon,
+  disabled,
```

Comment:
Query: What if we need to pass different styles to this component?
Add a prop named styles which holds the input value of styles